spec/javascripts/StrategiesSpec.js

```diff
@@ -24,33 +24,31 @@ describe("Strategies", function() {
 
   afterAll(function() {
     $("#test_body").remove();
+    $("#viewers_list").remove();
     $("body").removeClass("strategies new");
   });
 
-	it("test all viewers not checked", function() {
-    // all are unchecked
+	it("has selected all allies who can view the strategy when \"Select all\" is selected",  function() {
     newOrEdit.and.returnValue(true);
 
-    onReadyStrategies();
+    onReadyMomentsAndStrategies();
 
-    expect($(":checkbox[name='strategy[viewers][]']").prop("checked")).toBe(false);
-    expect($(":checkbox[id='viewers_all']").prop("checked")).toBe(false);
-  	expect($('#viewers_label').text()).toBe(ALL_ALLIES);
-  });
+    $('#viewers_all').click();
```

Comment:
You can also test ` expect($(":checkbox[name='strategy[viewers][]']").eq(0).prop("checked")).toBe(false);` before the click.